configs/Base-RRCNN-FPN.yaml

```diff
@@ -0,0 +1,54 @@
+MODEL:
```

Comment:
Please inherit from `Base-RCNN-FPN.yaml` as much as possible.
And add a comment here that this is just an example of what may need to be changed, and it is NOT a config meant for training directly (since we don't have code for a rotated dataset).

Reply:
I agree with the first part.
I don't think a comment is necessary. We never use the `Base-` configs to train directly.